Examples/Scripts/Python/full_chain_itk.py

```diff
@@ -33,15 +33,15 @@
 from acts.examples.itk import itkSeedingAlgConfig
 
 s = acts.examples.Sequencer(events=100, numThreads=-1)
-s = addParticleGun(
+addParticleGun(
     s,
     MomentumConfig(1.0 * u.GeV, 10.0 * u.GeV, True),
     EtaConfig(-4.0, 4.0, True),
     ParticleConfig(1, acts.PdgParticle.eMuon, True),
```

Comment:
```suggestion
    MomentumConfig(1.0 * u.GeV, 10.0 * u.GeV, transverse=True),
    EtaConfig(-4.0, 4.0, uniform=True),
    ParticleConfig(1, acts.PdgParticle.eMuon, randomizeCharge=True),
```
can you add kwargs as for the odd example? There are also a few other cases which could be changed if you like, but these are the important ones I think.